RELEASE.md

```diff
@@ -23,4 +23,11 @@ Keras pruning API:
 
  * Tested against TensorFlow 1.14.0, 2.0.0, and nightly.
 
-
+Keras clustering API:
+
+* New API for clustering of weights
```

Comment:
clustering of weights -> weight clustering

Reply:
I think both work fine, but _clustering of weights_ seems to be more precise to me, but it's rather subjective, isn't it?

Reply:
I will change it to _weight clustering_ to be more consistent with the documentation.